src/Microsoft.OpenApi.OData.Reader/Operation/RefPostOperationHandler.cs

```diff
@@ -66,7 +67,8 @@ protected override void SetResponses(OpenApiOperation operation)
         {
             OpenApiSchema schema = new OpenApiSchema
             {
-                Type = "String" // What to return?
+                Type = "object",
+                AdditionalProperties = new OpenApiSchema { Type = "object" }
```

Comment:
AdditionalProperties should only be applied to request body via `SetRequestBody` and not response.
```suggestion
                Type = "string"
```

Reply:
@peombwa  Did you intentionally change object back to string?

Reply:
Great catch, it should be type object. AdditionalProperties should only be applied to request bodies.